Solution1/Solution1.Tests/Infrastructure/Menu.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Weather.Tests.Infrastructure
+{
+    public static class Menu
+    {
+        public static string GetMenuRepresentation()
+        {
+            return "Select menu item:" +
+                "\r\n0 - Exit" +
```

Comment:
use Environment new line, \r\n different for different OS

Reply:
replace in all places in code

Reply:
+